### DISCUSSION TITLE
Offering Free Hosting for the Laragon Website

### DISCUSSION THREAD

**Discussion by alesanGreat** [leokhoa/laragon#580] [General]
I own a small hosting company. Sadly, I've noticed that the Laragon website has been taken down due to financial issues. I am in a position to provide full hosting for free as a way to sponsor such incredible work. Contact me for details at alesanGreat@gmail.com

**Comment by duongthanhthai** [comment]
It has lost a lot of documents and questions from the community. Very respectful but regretful
